services/ui-src/src/components/pages/Profile/ProfilePage.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
Just to note: we should be using userEvent instead of fireEvent. This is one that Ben caught on my last PR.

This was his comment on fireEvent:

> It's a good idea to use userEvent instead of fireEvent pretty much everywhere. fireEvent is a more low-level interface, whereas userEvent does a better job of simulating actual user flow.
> 
> It is in a separate package*, so if HCBS doesn't already have @testing-library/user-event, this would be a good time to install it.
> 
> *For now. The author (of both libraries) has said he intends to eventually include userEvent in the main package. For this and more, see his blog: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event

Reply:
I wrote a [ticket](https://jiraent.cms.gov/browse/CMDCT-3895) to go back and tackle some of the tests feedback from Ben. No changes necessary here.